.gitignore

```diff
@@ -2,3 +2,4 @@ _site
 .sass-cache
 .jekyll-metadata
 .DS_Store
+node_modules
```

Comment:
This helped took us from 11k down to 7.3k.  Still lots of files in the `themes/` which looks like it is sub-linked as a github-submodule.  I don't know enough about Hugo to know if those are all-required?  Or those could be sourced as part of the envrionment setup.

Reply:
let me find out!

Reply:
You should be able to gitignore the docsy theme folder, since this usually pulls in with a recursive submodule pull during the clone operation.

Reply:
thank you! I will run a diff first to make sure I didn't "improve" anything in the standard theme 😂